unlock-app/src/constants.ts

```diff
@@ -18,7 +18,7 @@ export const ETHEREUM_NETWORKS_NAMES = {
  * @param title
  * @returns {string}
  */
-export const pageTitle = title => {
+export function pageTitle(title: string): string {
```

Comment:
more a question than anything else. Is this valid TS too?

```ts
export const pageTitle = (title: string): string => {
```

Reply:
It is, and I really should have kept a consistent type of declaration between them.